schemas/fingerprint-server-api.yaml

```diff
@@ -714,6 +760,24 @@ components:
                   description: >
                     `true` if the request IP address is a known tor exit node, `false` otherwise.
                   example: false
+            privacySettings:
+              title: WebhookSignalResponsePrivacySettings
+              type: object
+              properties:
+                result:
+                  type: boolean
+                  description: >
+                    `true` if the request is from privacy aware browser (e.g. Brave with fingerprinting blocking), `false` otherwise.
```

Comment:
Are we intentionally wage or could we list all combinations of browsers and modes that the signal would evaluate as `true`?

Reply:
I did it intentionally to not share what we can detect and what not. @leenasudhakar what do you think?

Reply:
I would not list all the different methods used to compute the boolean value, but I would like to rephrase the description as 
```suggestion
                    `true` if the request is from a privacy aware browser (e.g. Tor) or from a browser in which fingerprinting is blocked. Otherwise `false`.
```

Reply:
done